src/WebUI/Common/KubeSummary.tsx

```diff
@@ -219,17 +249,24 @@ export class KubeSummary extends BaseComponent<IKubeSummaryProps, IKubernetesCon
 
     private _getMainContent(): JSX.Element {
         const pageContent = this.state.resourceSize > 0 ? this._getMainPivot() : this._getZeroData();
+        const headerProps: IHeaderProps = {
+            title: this.props.title,
+            titleSize: TitleSize.Large,
+            className: "content-main-heading",
+            description: this.props.clusterName
+                ? localeFormat(Resources.SummaryHeaderSubTextFormat, this.props.clusterName)
+                : localeFormat(Resources.NamespaceHeadingText, this.state.namespace || "")
+        };
+
+        if (this.props.onTitleBackClick) {
+            headerProps.titleIconProps = { iconName: "Back", onClick: this.props.onTitleBackClick, className: "cursor-pointer" };
```

Comment:
in future we may have to make this as button, today this button is not reachable via keyboard.

Reply:
I'll track this